API.md

```diff
@@ -121,7 +132,8 @@ refresh_count and max_refresh_count can be used to determine when a re-authentic
 
     + Headers
 
-            X-Vault-Token: 7f6808f1-ede3-2177-aa9d-45f507391310
+            X-Cerberus-Token: 7f6808f1-ede3-2177-aa9d-45f507391310
```

Comment:
Nit: it doesn't really matter but tokens no longer contain non-alphanumeric chars, no more hyphens.

Reply:
Updated